### ISSUE TITLE
only works in cpu mode , but gpu outputs nothing

### ISSUE THREAD

**Issue by yslion** [Softcatala/whisper-ctranslate2#26]
os: win11 , cuda 11.6   cudnn 8.9  rtx3060 4G 

every example  in readme  i've tried ,  if no --device  , no outputs ,  if  --device cpu   it works  but slow.

and  I've tried  the newest version   

 whisper-ctranslate2-0.1.9
 whisper-ctranslate2-0.2.0

**Comment by jordimas** [comment]
Does it make any difference if you use base, tiny, small or large-v2 models?

**Comment by yslion** [comment]
> large-v2

 large-v2 is still downloading ,   but the other 3  , shows nothing 

(venv) D:\py\whisper-ctranslate2>whisper-ctranslate2 1.mp3 --model medium
Detecting language using up to the first 30 seconds. Use `--language` to specify the language
Estimating duration from bitrate, this may be inaccurate

(venv) D:\py\whisper-ctranslate2>whisper-ctranslate2 1.mp3 --model base  
Detecting language using up to the first 30 seconds. Use `--language` to specify the language
Estimating duration from bitrate, this may be inaccurate

(venv) D:\py\whisper-ctranslate2>whisper-ctranslate2 1.mp3 --model small
Detecting language using up to the first 30 seconds. Use `--language` to specify the language

**Comment by Zacharie-Jacob** [comment]
when you say output, do you mean specifically not saving the files out?  Does it display to the termianl, though?

There is this issue about display but no saved files:

https://github.com/jordimas/whisper-ctranslate2/issues/11

**Comment by zx3777** [comment]
display to the termianl but not save , everytime i use this. I have no choice but to use the original version, which is slow but no problem.

1.0 could save，but when i use --word_timestamps=ture ，the  --highlight_words=true will auto turn on，and cant be closed.

**Comment by yslion** [comment]
@Zacharie-Jacob 
no,  'no outputs' I said , means  no  display about translating  ,  no save files 

this is my cmd line 
`whisper-ctranslate2 1.mp3 --model medium`

this is it's  output  
```
Detecting language using up to the first 30 seconds. Use --language to specify the language
Estimating duration from bitrate, this may be inaccurate
```
and then  quit   without  showing anything

**Comment by efonte** [comment]
The problem is in the `faster_whisper` package. If you run for example this script it will show you what the problem is.

```python
from faster_whisper import WhisperModel

model_size = "large-v2"
model = WhisperModel(model_size, device="cuda", compute_type="float16")
segments, info = model.transcribe("audio.wav", beam_size=5)
print("Detected language '%s' with probability %f" % (info.language, info.language_probability))
for segment in segments:
    print("[%.2fs -> %.2fs] %s" % (segment.start, segment.end, segment.text))
```

```
PS C:\> python .\script.py
Could not load library cudnn_cnn_infer64_8.dll. Error code 126
Please make sure cudnn_cnn_infer64_8.dll is in your library path!
```
The error has been solved by installing zlib:
https://docs.nvidia.com/deeplearning/cudnn/install-guide/index.html#install-zlib-windows

Maybe it would be good to show the error in `whisper-ctranslate2` since it hides it.

**Comment by yslion** [comment]
> The problem is in the `faster_whisper` package. If you run for example this script it will show you what the problem is.
> 
> ```python
> from faster_whisper import WhisperModel
> 
> model_size = "large-v2"
> model = WhisperModel(model_size, device="cuda", compute_type="float16")
> segments, info = model.transcribe("audio.wav", beam_size=5)
> print("Detected language '%s' with probability %f" % (info.language, info.language_probability))
> for segment in segments:
>     print("[%.2fs -> %.2fs] %s" % (segment.start, segment.end, segment.text))
> ```
> 
> ```
> PS C:\> python .\script.py
> Could not load library cudnn_cnn_infer64_8.dll. Error code 126
> Please make sure cudnn_cnn_infer64_8.dll is in your library path!
> ```
> 
> The error has been solved by installing zlib: https://docs.nvidia.com/deeplearning/cudnn/install-guide/index.html#install-zlib-windows
> 
> Maybe it would be good to show the error in `whisper-ctranslate2` since it hides it.

thank you for your reply , I ve  fix the zlib issue (see  above  ,line 5  )  . 



the matter is , live_transcribe   ,  print out 'Transcribing..'   and the quit without any outputs
 ```
(venv) D:\py\whisper-ctranslate2>whisper-ctranslate2 --live_transcribe True --language en  --model tiny 
Listening.. (Ctrl+C to Quit)
...............................................................................................
Transcribing..
...........
(venv) D:\py\whisper-ctranslate2>
```
and  @jordimas

**Comment by jordimas** [comment]
This has been fixed with version 0.26
